services/middlewareServices/AuthMiddlewareService.js

```diff
@@ -77,7 +81,7 @@ class AuthMiddlewareService {
         logger.error(`Authentication error: JWT token expired. Url: ${url}`)
         throw new AuthError(`JWT token has expired`)
       } else {
-        logger.error(`Authentication error. Url: ${url}`)
+        logger.error(`Authentication error. Message:${err.message} Url: ${url}`)
```

Comment:
also extremely minor nit: could we add a space like so `Message: ${err.message}`

Reply:
Fixed in https://github.com/isomerpages/isomercms-backend/pull/423/commits/68574b98a088437c7edca179a1930164b22a797c